torchmultimodal/modules/layers/normalizations.py

```diff
@@ -20,3 +21,56 @@ def forward(self, x):
             self.eps,
         )
         return output.type_as(x)
+
+
+class FrozenBatchNorm2d(nn.Module):
```

Comment:
is this very mdetr resnet specific? if yes, consider moving to that module

Reply:
It's also used in the ResNet benchmark from FasterRCNN, for example. But not sure how widespread the usage is in general. I think moving it to the MDETR ResNet module should be reasonable